src/spotbugs/excludesFilter.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
```

Comment:
Seems easier in the short term to temporarily set the SpotBugs threshold to "high" in `pom.xml` than to introduce this exclusion file which would then need to be maintained. No strong preference, though.